setup.cfg

```diff
@@ -82,11 +82,73 @@ setup_requires =
     wheel
 zip_safe = False
 
+[options.packages.find]
```

Comment:
do we really need this? it's already defined in manifest.in file and it works well

Reply:
yep, it is the new recommendation from setuptools

Reply:
let's remove it here if it's not required for the PR. I want to do some testing of the distribution built before we add this.